### ISSUE TITLE
concatenate multiple datasets

### ISSUE THREAD

**Issue by lqzhao** [zylo117/Yet-Another-EfficientDet-Pytorch#501]
I got several COCO-format datasets, but this repo does not seem to leave an interface to concatenate multiple datasets or something to form a dataset_list. If someone has done this personally, please share the code or contribute to this repo~ That would be great!

**Comment by Pari-singh** [comment]
Hi @Iqzhao, want to understand, are you looking for a script where you could train datasets which might have differences, let's say in image size etc? If no, then would you not try concatenating the labels and adding all the images to the same folder? Please correct my understanding if I am wrong.

**Comment by lqzhao** [comment]
Thanks for your reply! I mean I have collected several COCO-format datasets, in different folders, like COCO2017, COCO2014... It's not convenient to put all data in the same folder which means you should enrich ANNOTATION file every time new data joins. Fortunately, I've sovled this problem by a Pytorch Class named _torch.utils.data.ConcatDataset(datasets)_, that makes concatnating datasets possible!  I'll close this issue.
